Backpacks.cs

```diff
@@ -666,6 +708,29 @@ public ushort BackpackSize
 
             [JsonProperty("Blacklisted Items (Item Shortnames)")]
             public HashSet<string> BlacklistedItems;
+
+            [JsonProperty(PropertyName = "GUI button")]
+            public GUIButton GUI = new GUIButton();
+
+            public class GUIButton
+            {
+                [JsonProperty(PropertyName = "Enabled?")]
+                public bool Enabled = true;
+
+                [JsonProperty(PropertyName = "Image")]
+                public string Image = "https://i.imgur.com/CyF0QNV.png";
+
+                [JsonProperty(PropertyName = "Background color (RGBA format)")]
+                public string Color = "1 1 1 0.15";
+
+                [JsonProperty(PropertyName = "AnchorMin")]
+                public string AnchorMin = "0.6445 0.025";
+                //Left side button = "0.2945 0.025"
```

Comment:
It would probably be better to document the left side values in the README. Otherwise, people won't see it unless they look in the code is too advanced for most people.